libseccomp/src/api.rs

```diff
@@ -46,6 +54,15 @@ pub fn set_api(level: u32) -> Result<()> {
 ///
 /// If the API level can not be detected due to the library being older than v2.4.0,
 /// an error will be returned.
+///
+/// # Examples
+///
+/// ```
+/// # use libseccomp::*;
+/// set_api(1)?;
+/// assert_eq!(1, get_api()?);
```

Comment:
To be consistent with the std-lib.

```suggestion
/// assert_eq!(get_api()?, 1);
```

Reply:
Added the fix in [commit](https://github.com/libseccomp-rs/libseccomp-rs/commit/2e07e17f07f92e2b6f7766c3c2d67495dc762efe)